capstone/css/style.css

```diff
@@ -0,0 +1,978 @@
+}
+
+.titleWrap {
+  display: flex;
+  flex-direction: column;
+}
+
+header {
+  position: fixed;
+  z-index: 5;
+  width: 100%;
+}
+
+.menu .close {
+  display: none;
+}
+
+#artistSection {
+  display: flex;
+  flex-direction: column;
+}
+
+#artistSection .holder {
+  display: flex;
+  flex-direction: column;
+  order: 0;
+}
+
+#featuredArtists {
+  order: 1;
+}
+
+.artist {
+  display: flex;
+  flex-direction: row;
+}
+
+.artistDetail {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.seeMore {
+  display: none;
+}
+
+footer {
+  flex-direction: column;
+}
+
+.imgHolder {
+  display: flex;
+  width: 48%;
+  align-items: flex-end;
+  justify-content: end;
+  background-image: url("../images/scratched.jpg");
+  background-repeat: no-repeat;
+  background-size: 50% 50%;
+}
+
+.artist span {
+  font-style: italic;
+  color: #ec5242;
+  line-height: 0.8rem;
+}
+
+.artist .artistHolder {
+  display: flex;
+  flex-direction: column;
+  gap: 0.3rem;
+}
+
+.artist .underLine {
+  height: 0;
+  width: 10%;
+  border-bottom: 1px solid #d3d3d3;
+}
+
+@media only screen and (max-width: 768px) {
+  .main .home {
+    padding: 5rem 1rem 4rem;
+    gap: 1rem;
+  }
+
+  .mainProgram .title {
+    -webkit-text-stroke-width: 0.2px;
+    -webkit-text-stroke-color: black;
+    font-size: 1.2rem;
+    font-weight: lighter;
+  }
+
+  .homeContent .home {
+    gap: 1rem;
+    padding-top: 1.5rem;
+  }
+
+  .homeContent .description {
+    padding: 1rem;
+    font-size: 0.8rem;
+    border: 1px solid rgba(255, 255, 255, 0.9);
+  }
+
+  .home .greet {
+    font-size: 1.5rem;
+  }
+
+  .about .greet {
+    display: block;
+    font-size: 1.5rem;
+  }
+
+  .location {
+    font-size: 0.8rem;
+  }
+
+  main .about {
+    gap: 2rem;
+    padding: 5rem 1rem 2rem;
+  }
+
+  .programBtn {
+    display: flex;
+    font-weight: lighter;
+    margin: 0 9%;
+  }
+
+  .about .description {
+    line-height: 165%;
+    padding: 1.5rem 1rem;
+  }
+
+  .about .contact {
+    line-height: 160%;
+    padding: 0 3rem;
+  }
+
+  .creativeLogo {
+    padding: 1rem 1rem 3rem;
+  }
+
+  .creativeLogo .holder {
+    padding: 1rem 4rem;
+  }
+
+  .logoDesc {
+    padding: 0 3rem;
+    line-height: 150%;
+  }
+
+  .aboutLogo {
+    margin-top: 2rem;
+    border: 1px solid #d3d3d3;
+  }
+
+  .mainProgram {
+    gap: 2rem;
+    padding: 2rem 1rem;
+  }
+
+  .pastEvents {
+    flex-direction: column;
+    justify-content: center;
+    padding: 2rem 11%;
+    gap: 1rem;
+  }
+
+  .event {
+    padding: 5rem 3rem;
+    gap: 1rem;
+  }
+
+  .year {
+    font-size: 2rem;
+  }
+
+  .name {
+    font-size: 1rem;
+  }
+
+  .programDetails {
+    display: none;
+  }
+
+  .partners {
+    flex-direction: column;
+    padding: 2rem 1rem;
+    height: 70%;
+    gap: 1rem;
+  }
+
+  .aboutPartners {
+    display: flex;
+    flex-direction: column;
+    padding: 2rem 1rem;
+    height: 70%;
+    gap: 1rem;
+  }
+
+  .partners .title {
+    color: #fff;
+    font-size: 1.5rem;
+  }
```

Comment:
- [ ] Please kindly implement the **color** as we have it in the design for the mobile version as well, I observed that you perfectly did for the desktop version. Working perfectly with a given template always shows that we pay attention to details. 👍 

## Your Design
![DSC_7491](https://user-images.githubusercontent.com/103386183/218038942-aace7f74-d527-4b1c-ac3e-8dcb22c7943b.JPG)